CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Changelog
 
 [![PyPI version](https://badge.fury.io/py/iotedgehubdev.svg)](https://badge.fury.io/py/iotedgehubdev)
+## 0.12.0 - 2019-11-xx
+### Added
+* Add cert extensions required by iot edge transparent gateway
```

Comment:
It's not added. How about move to Changed just say "Update cert generation logic"?

Reply:
OK